src/core/client.ts

```diff
@@ -510,6 +510,19 @@ export class Client<M = Indexable> implements Observable<ClientEvent<M>> {
       return Promise.resolve();
     }
 
+    const getPeers = (
+      peersMap: Record<string, Record<string, M>>,
+      key: string,
+    ) => {
+      const attachment = this.attachmentMap.get(key);
+      const peers: Record<string, M> = {};
+      for (const [key, value] of attachment!.peerPresenceMap!) {
+        peers[key] = value.data;
+      }
+      peersMap[key] = peers;
+      return peersMap;
+    };
+
```

Comment:
Can we extract `getPeers` from `handleWatchDocumentsResponse` and use them together?

Reply:
Inside client.ts Client class, there is already another function named getPeers. Would it be OK to create a new method inside the class with a different name other than getPeers?